test/controllers/admin/assignees_controller_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+class Admin::AssigneesControllerTest < ActionController::TestCase
+  attr_reader :user, :application, :lawyer
+
+  setup do
+    @user = users(:admin)
+    @application = applications(:in_progress)
+    @lawyer = users(:lawyer)
+
+    sign_in_as(@user)
+  end
+
+  test "POST #create associates lawyer to application and makes primary" do
+    assert_difference 'application.assignees.count' do
+      post :create, assignee: { application_id: application.id, user_id: lawyer.id }
+    end
+
+    assert application.assignees.where(user: lawyer).exists?
+
+    assert_redirected_to admin_application_path(application)
+    assert_equal 'Successfully assigned lawyer.', flash[:notice]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.